.travis.yml

```diff
@@ -39,8 +39,11 @@ matrix:
     - name: "make depscheck"
       script: make depscheck
     - name: "make website-lint"
-      script: make website-lint
+      script:
+      - curl -sLo /tmp/terraform.zip https://releases.hashicorp.com/terraform/0.12.18/terraform_0.12.18_linux_amd64.zip
+      - unzip /tmp/terraform.zip -d /tmp
+      - mv /tmp/terraform ~/bin
+      - export PATH="~/bin:$PATH"
```

Comment:
cc @kmoe since you've been looking into using TF Core as a binary too

Reply:
I really wish there was a better way of doing this but this was quick and easy